packages/browser-ppx/tests/at_platform.t

```diff
@@ -124,6 +124,71 @@ Pstr_value
   $ ./standalone.exe -impl input_let.ml -js | ocamlformat - --enable-outside-detected-project --impl
   let x = 42 [@@platform js]
 
+Pexp_let
+
+  $ cat > input_let.ml << EOF
+  > let x =
+  >   let y = 42 [@@platform js] in
+  >   let y = 44 [@@platform native] in
+  >   y
+  > EOF
+
+  $ ./standalone.exe -impl input_let.ml | ocamlformat - --enable-outside-detected-project --impl
+  let x =
+    let y = 44 [@@platform native] in
```

Comment:
platform attribute should not be part of the output right?

Reply:
The platform attr is part of the output on all the examples of this test. Like this one:
```
Nested

  $ cat > input.ml << EOF
  >  module X = struct
  >    include struct
  >    type t = Js.Json.t
  >    let a = 2 + 2
  >    end [@@platform js]
  >  
  >    include struct
  >      type t = Js.Json.t
  >      let a = 4 + 4
  >    end [@@platform native]
  >  end
  > EOF

With -js flag it picks the block with `[@@platform js]`

  $ ./standalone.exe -impl input.ml -js | ocamlformat - --enable-outside-detected-project --impl
  module X = struct
    include struct
      type t = Js.Json.t
  
      let a = 2 + 2
    end [@@platform js]
  end

Without -js flag, it picks the block with `[@@platform native]`

  $ ./standalone.exe -impl input.ml | ocamlformat - --enable-outside-detected-project --impl
  module X = struct
    include struct
      type t = Js.Json.t
  
      let a = 4 + 4
    end [@@platform native]
  end
```